src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindIntegrationTest.java

```diff
@@ -2152,6 +2032,269 @@ public void notWithComparisonExpressionAndLogicalExpression() {
     }
   }
 
+  @Nested
+  @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+  @Order(2)
+  class FindWithProjection {
+    @Test
+    @Order(1)
+    public void insertData() {
+      insert(
+          """
+                            {
+                              "insertOne": {
+                                "document": {
+                                  "_id": "doc6",
+                                  "price.usd": 5
+                                }
+                              }
+                            }
+                          """);
+    }
+
+    @Test
+    public void byIdWithProjection() {
+      String json =
+          """
+                          {
+                            "find": {
+                              "filter" : {"_id" : "doc1"},
+                              "projection": { "_id":0, "username":1 }
+                            }
+                          }
+                          """;
+
+      String expected =
+          """
+                  {"username":"user1"}
+                  """;
+      given()
```

Comment:
Forgot to mention earlier, but for new tests (which this may or not be), we can use helper methods from `AbstractCollectionIntegrationTestBase`: for passing (non-error) calls

```
    givenHeadersPostJsonThenOkNoErrors(json)
       .body("$", ...)
```

and error cases

`givenHeadersPostJsonThenOk(json)`

that makes tests bit less verbose and easier to read (can focus on actual request and response).